flink-yarn/pom.xml

```diff
@@ -85,8 +85,20 @@ under the License.
 			<artifactId>guava</artifactId>
 			<version>${guava.version}</version>
 		</dependency>
-		
-		
+
+		<dependency>
+			<groupId>com.typesafe.akka</groupId>
+			<artifactId>akka-testkit_${scala.binary.version}</artifactId>
+			<scope>test</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-runtime_${scala.binary.version}</artifactId>
```

Comment:
Here's the culprit. This has to be `flink-runtime_2.10` because it is a Flink artifact. The `scala-2.11` property only affects Scala dependencies of Flink artifacts. The Flink artifacts names remain the sames unless changed via the `change-scala-version.sh` script.
 
If I remember correctly we don't use properties in artifact names because they are not resolved by Maven.